linux/start/ibm/adoc/gs_sles_ibm-hpvs.adoc

```diff
@@ -1553,11 +1553,140 @@ slebci-paynow-website compose-paynow-1 info GET /api/v1/transactions
+
+The recommended method to access the attestation document is via the workload.
+The https://cloud.ibm.com/docs/vpc?topic=vpc-financial-transaction-confidential-computing-on-hyper-protect-virtual-server-for-vpc[PayNow Node.js application] application developers added API calls that provide an https://github.com/ibm-hyper-protect/paynow-website/blob/main/app/app.js#L42[encrypted] or an https://github.com/ibm-hyper-protect/paynow-website/blob/main/app/app.js#L47[unencrypted] attestation document.
+This guide will provide the steps to access the encrypted attetation document and verify the attestation data of a deployed PayNow Node.js application.
+
+. Enable the workload to access the attestation data.
+
+.. Add `volumes` section to `docker-compose.yml`.
+For example if you are using the pre-built PayNow Node.js application container image.
++
+[source, yaml]
+----
+services:
+  paynow:
+    image: ghcr.io/mfriesenegger/slebci-paynow-website@sha256:ecb229f68aef81ca4a2b7b5a9eb192081fa2a170e44d9e5a28180bb12682ce5d
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+    volumes:
+      - "/var/hyperprotect/:/var/hyperprotect/:ro"
+----
+
+.. Recreate the base64 encoded tar archive.
+
+.. Update the `archive` item in the workload contract with the new base64 data.
+
+. Use a private/public key pair for attestation document encryption.
+
+.. In the directory where the Docker Compose file exists, the following commands will create the private/public key pair for attestation encryption.
++
+[NOTE]
+====
+You will be asked to define a pass phrase when creating the private key as well as provide the same pass phrase when creating public portion of the key. 
+====
++
+[source, console]
+----
+openssl genpkey -aes256 -algorithm RSA -pkeyopt rsa_keygen_bits:2048 -out attest-private-key.pem
+openssl pkey -in attest-private-key.pem -out attest-public-key.pem -pubout
+----
+
+.. Add `attestationPublicKey` to the contract.
++
+[NOTE]
+====
+A portion of the contract file is shown.
+====
++
+[source, yaml]
+----
+workload: |
+  type: workload
+  compose:
+    archive: H4sIAAAAAAAAA+3UzW7bMAwHcJ/zFELucahP2gYG7LjjXkG2mNRoXQdWtrZvP6XBsCyHboeuW7H/ 
+attestationPublicKey: <attest-pubkey> <1>
+----
++
+<1> where the output of the following command replaces the *<attest-pubkey>*.
++
+`IFS=$'\n'; echo -n \"; for line in $(cat attest-public-key.pem); do echo -n $line"\n"; done; echo \"`
+
+. Deploy the workload either in IBM Cloud or on-premises.
+
+. Enter https://<ip address>:8443//api/v1/attestation to access the encrypted attestation document.
++
+The floating IP address noted earlier for the VPC deployment in IBM Cloud.
++
+The IP address assigned to the HPCR VM running on the KVM host.
+
+. Save the encrypted attestation document to `se-checksums.txt.enc`.
+The file can be save on the local system where the Docker Compose file is located.
```

Comment:
Insert between the "Save the encrypted ..." line and the "The file can be saved ..." line the following two lines:
//
+

This will force the second sentence to a new line.